cfme/infrastructure/config_management/__init__.py

```diff
@@ -375,7 +375,7 @@ def update(self, updates, cancel=False, validate_credentials=False):
         else:
             view.entities.save.click()
             view.flash.assert_success_message(
-                '{} Provider "{}" was updated'.format(self.ui_type, updates['name'] or self.name))
```

Comment:
I'm guessing you were hitting a `KeyError` here?  I think this needs to remain, since the update to `self.name` happens after this assertion.

**Required** Please use `updates.get('name')` here to avoid the `KeyError`. 

**Optional** As a followup PR, move these flash message string assertions to a CRUD testcase, and only use `view.flash.assert_no_error` within this method.

Reply:
Yes, I was hitting a KeyError. Updated the PR to use updates.get('name') here to avoid the KeyError.